beacon-chain/sync/data_columns_sampling.go

```diff
@@ -538,9 +538,11 @@ func verifyColumn(
 		return false
 	}
 
-	vf := columnVerifier(roDataColumn, verification.SamplingColumnSidecarRequirements)
+	// TODO: Once peer sampling is used, we should verify all sampled data columns in a single batch.
+	verifier := dataColumnsVerifier([]blocks.RODataColumn{roDataColumn}, verification.SamplingColumnSidecarRequirements)
```

Comment:
This is a nit, but in go its always more preferable for shorter variable names (vf vs verifier). This was done in a few places in this PR